src/protocol/control_layer/resend_response/ResendResponseResent.js

```diff
@@ -0,0 +1,25 @@
+import ControlMessage from '../ControlMessage'
+import { validateIsNotEmptyString, validateIsNotNegativeInteger } from '../../../utils/validations'
+
+const TYPE = 5
```

Comment:
I do wish rather than having each file define their type number themselves all these `TYPE` keys came from a single, shared named map e.g. `TYPE = {ResendResponse: 5}` to be used like `TYPE.ResendResponse`. Can still export as `ResendResponse.TYPE` as well. Perhaps for another refactoring.